sleap/config/pipeline_form.yaml

```diff
@@ -424,10 +424,6 @@ inference:
       This tracker "shifts" instances from previous frames using optical flow
       before matching instances in each frame to the <i>shifted</i> instances from
       prior frames.'
-    # - name: tracking.max_tracking
-    #   label: Limit max number of tracks
-    #   type: bool
-      default: false
```

Comment:
Hi @gitttt-1234, thanks for reviewing the PR!

This part is just about removing commented code anyway.

Reply:
sure, but we still need to set flow to `false` by default

Reply:
I think this "default: false" was supposed to be commented out because it belonged to the "tracking.max_tracking" option. As it was, it was part of the "text" block explaining what Tracking with optical flow is, so it was just ignored.
That is why I also removed it.

"tracking.tracker" default value is set at line 399:
```
- name: tracking.tracker
  label: Tracker (cross-frame identity) Method
  type: stacked
  default: none
  options: none,flow,simple
```